views/Covid/covid.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="covid.css" />
+  </head>
+  <body>
+    <div class="left">
+      <h1 class="page-title">Covid Stuff</h1>
+
+      <div class="form-container">
+        <form class="radio-button-wrapper" id="work">
+          <p class="question">How would you prefer to work?</p>
+
+          <div>
+            <input type="radio" id="office" name="work" value="office" />
+            <label>From Sairme Office</label>
```

Comment:
label ტეგს უნდა ქონდეს for ატრიბუტი რომელიც დაემთხვევა შესაბამისი input-ის id-ს
მაგალითად აქ უნდა გქონდეს
```
<label for="office">From Sairme Office</label>
```
შედეგად მოხდება ის რომ ამ ლაბელზე დაჭერისას ფოკუსი მოხდება ინფუთზე